### PR TITLE
feat(kms): use kms for also libplanet side

### LINKED ISSUE #47: feat(kms): support kms for libplanet side
The bridge already supports AWS KMS for ethereum side but not libplanet side yet. It should support libplanet side also.

### PR DESCRIPTION
**Pull request by moreal**
This pull request resolves #47

### CODE REVIEW

File: bridge/src/kms-ncg-signer.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      const otherSBigInt = nBigInt - sBigInt;
+      const otherS = bigintToNodeBuffer(otherSBigInt);
+      const sequence = new Sequence();
+      // @ts-ignore
+      sequence.valueBlock.value.push(new Integer({ isHexOnly: true, valueHex: r }));
+      console.log("n", nBigInt)
+      console.log('r', bufToBigint(r));
+      console.log('s', sBigInt);
+      console.log('otherS', otherSBigInt);
+      console.log('use otherS', sBigInt > otherSBigInt);
```
Comment (longfin) [inline bridge/src/kms-ncg-signer.ts:71]:
```suggestion
```

File: bridge/src/kms-ncg-signer.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        sequence.valueBlock.value.push(new Integer({ isHexOnly: true, valueHex: otherS }).convertToDER());
+      }
+      else {
+        // @ts-ignore
+        sequence.valueBlock.value.push(new Integer({ isHexOnly: true, valueHex: s }).convertToDER());
+      }
+
+      const sequence_buffer = toBuffer(sequence.toBER(false));
+      console.log('original sign: ', asn1Signature.toString('hex'));
+      console.log('sign: ', sequence_buffer.toString('hex'));
```
Comment (longfin) [inline bridge/src/kms-ncg-signer.ts:83]:
```suggestion
```

File: bridge/test/aws-dependent/sign.aws.spec.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        it("should be sign with correct key.", async () => {
+            const base64String = await mockHeadlessGraphQlClient.createUnsignedTx("", "");
+            const base64Buffer = Buffer.from(base64String, "base64");
+            const digest = Buffer.allocUnsafe(32).fill(0);
+            base64Buffer.copy(digest);
+
+            const sign = await signer.sign(digest);
+
+            const publicKey = await signer.getPublicKey();
+            console.log(publicKey.toString('base64'));
```
Comment (riemannulus) [inline bridge/test/aws-dependent/sign.aws.spec.ts:56]: We need to verify the sign here.

**Review by longfin — CHANGES_REQUESTED** [review]
It seems that headless is failing to verify the signature from AWS KMS yet. 😢

**Comment by moreal** [comment]
> 
> 
> It seems that headless is failing to verify the signature from AWS KMS yet. 😢

I don't know your situation fully but I also got situation like you while testing (i.e. running `yarn test`).
I cleaned up code correctly and I believed the mock values are valid. But they were wrong so I updated the values in 9b08973. Could you check it again?

P.S. I'm running endless-loop `yarn test` to try catching edge case error.

File: bridge/test/aws-dependent/sign.aws.spec.ts (hunk 3 above)
Comment (moreal) [reply bridge/test/aws-dependent/sign.aws.spec.ts:56]: In my thought, this test is useless because the *bridge* application is using `KMSNCGSigner` which implemented by this projecct without `KmsSigner`  implementation. How do you think about this?

**Review by riemannulus — APPROVED** [review]
